sys/include/crypto/modes/cbc.h

```diff
@@ -7,11 +7,12 @@
  */
 
 /**
+ * @defgroup    sys_crypto_mode_cbc Cipher block chaining mode
```

Comment:
This is really completely unrelated. These files are perfectly fine to all be grouped in `sys_crypto`. If you disagree, please move this group introductions to a separate PR.